src/Breadcrumb/__tests__/Breadcrumb.tests.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Breadcrumb from '../Breadcrumb';
+import { BrowserRouter } from 'react-router-dom';
+
+describe('<Breadcrumb />', () => {
```

Comment:
Let's make the test data more generic -- `/page/subpage` instead of `pipeline/tasks`, etc.